cfgov/v1/blocks.py

```diff
@@ -164,12 +169,32 @@ class Meta:
 
 
 class PlaceholderFieldBlock(blocks.FieldBlock):
+    """
```

Comment:
Instead of making numerous changes to support PlaceholderFieldBlock/PlaceholderCharBlock, can we just deprecate them and remove all of this code from the repository? As far as I can tell these blocks aren't used in any atomic elements or on any pages, so all of these changes are just to get tests to pass. Looking back it seems like these blocks were introduced in #2605 and then used in #2532 for the Wagtail-based Bureau structure; but now that the Bureau structure is no longer implemented as StreamFields, they seem to have been abandoned.

Reply:
Hmm, agreed. Let's rip them out! @jslay-excella, let me know if you need any help doing so.

Reply:
@chosak I think we'll leave it for now — because it's a lot of effort to rip the `Placeholder*` out of the old migrations, we're going to deal with in a separate PR.